src/kubernetes_upgrade.py

```diff
@@ -232,10 +232,12 @@ def determine_partition() -> int:
                 # also applicable `if not force`, but is unlikely to happen since all units are
                 # healthy `if not force`.
                 message = f"Attempting to upgrade unit {self._partition}"
+                action_event.set_results({"result": message})
+                logger.warning(f"Resume upgrade event succeeded: {message}")
             else:
                 message = f"Upgrade resumed. Unit {self._partition} is upgrading next"
-            action_event.set_results({"result": message})
-            logger.debug(f"Resume upgrade event succeeded: {message}")
+                action_event.set_results({"result": message})
+                logger.debug(f"Resume upgrade event succeeded: {message}")
```

Comment:
> Suggest leaving action_event as-is instead of duplicating

👍🏻 

> Suggest keeping the debug log for "resume upgrade event succeeded" since it might be a bit confusing to user to have a warning log with "succeeded"

The idea was to warn whenever a _potentially destructive_ action takes place. Given that at that particular path of the codebase the user is forcing an upgrade without checking for health conditions, that is one of the cases.

I could move the warning to the beginning of the method, to just warn whenever the `force` param is found within the `action_event` argument. Do you like that better?

Reply:
^ that's a good compromise

Reply:
Done in commit https://github.com/canonical/mysql-router-k8s-operator/pull/336/commits/9888ce75b998af14d365374f0d5b8c320663482b.